doc/api/path.md

```diff
@@ -452,10 +464,14 @@ On Windows:
 // returns ['foo', 'bar', 'baz']
 ```
 
+*Note*: This property is *not* read-only and may be modified.
```

Comment:
I'm not sure how useful this information is since the `path` module does not even use `path.sep` itself.